apps/blockchain/lib/blockchain/block.ex

```diff
@@ -734,7 +730,24 @@ defmodule Blockchain.Block do
       |> put_receipt(trx_count, receipt, db)
       |> put_transaction(trx_count, trx, db)
 
-    do_add_transactions(updated_block, transactions, db, trx_count + 1, config)
+    do_add_transactions(updated_block, transactions, db, trx_count + 1, chain)
+  end
+
+  defp create_receipt(block_header, new_state, total_gas_used, logs, tx_status, chain) do
```

Comment:
It seems receipts are not fully implemented yet. We are not using bloom filters. Also, I think we need to store them somewhere

Reply:
Yeah, receipts aren't fully implemented yet. There's an issue open for handling those -> https://github.com/poanetwork/mana/issues/66

Reply:
We should probably add an issue to implement/store receipts. I imagine that's a part of Frontier's implementation that we never got around to doing. I wouldn't want to add that in this PR, since this is just meant to add the transaction status of `0` or `1` to the receipt.